### DISCUSSION TITLE
Variable uninitialized error

### DISCUSSION THREAD

**Discussion by bcskillings** [RefindPlusRepo/RefindPlusUDK#9] [General]
I'm following the instructions on building on MacOS and came across an error. I'm pretty sure this is user error, not a bug, so I'm posting here rather than issues (please let me know if this is improper, I'm not familiar with the proper etiquette).

I've made it to "Build Required BaseTools C Tool Binaries" and when running
`make -C BaseTools/Source/C`
I get this error
`error: variable 'StringIndex2' is uninitialized when used here [-Werror,-Wuninitialized]`

I've poked around a bit online and found posts about a similar issue but I'm not quite sure how to implement it to this build. The full output is in the attached screenshot, there's a couple things that line up that I wouldn't be able to get proper here. 

<img width="1059" alt="Screen Shot 2021-04-04 at 6 17 28 AM" src="https://user-images.githubusercontent.com/31419498/113507023-b0e76400-950d-11eb-8f0a-569ba52c5b34.png">

**Comment by dakanji** [comment]
Not facing that issue but yes, StringIndex2 is indeed being called without being initialised.
So, doesn't seem to be a user error.

Will have a look for a fix. Thanks.

**Comment by dakanji** [comment]
New issue created: #10
Please follow up there. Thanks

**Comment by dakanji** [comment]
Fixed. Thanks

Update your repo and try again.
